.rubocop_fixme.yml

```diff
@@ -129,7 +129,7 @@ RSpec/ExampleLength:
     - 'spec/services/hyrax/workflow/state_machine_generator_spec.rb'
     - 'spec/services/hyrax/workflow/workflow_importer_spec.rb'
     - 'spec/views/**/*'
-
+    - 'spec/wings/valkyrie/persister_spec.rb'
```

Comment:
do we plan to fix this as this work develops, or does this belong in a different .rubocop config?

Reply:
Well, eventually we will remove all the triggering tests, because they'll be replaced with the call to the shared spec.

But mostly I put it here because in my experience you can't exclude in both `_todo` and `.rubocop.yml` for the same infraction; they just all go red. Granted I've never tried it in this codebase.